athena_cfn/README.md

```diff
@@ -0,0 +1,193 @@
+# Query Quilt Package Metadata in AWS Athena
+
+The code in this example shows you to enable (and test) use of the "Queries"
+pane of the Web Catalog for querying Quilt metadata for specific S3 bucket.
+
+### NOTE: Validating CloudFormation files
+
+If for any reason you need to edit the CloudFormation files,
+be sure to lint them before using:
+
+```
+$ pip3 install --upgrade taskcat cfn-lint
```

Comment:
Not a blocker but it would be nice to provide a requirments.txt instead of an inline install like this.